### DISCUSSION TITLE
How to echo text with custom color in nusll

### DISCUSSION THREAD

**Discussion by gipsyh** [nushell/nushell#5377] [Q&A] [answered]
How to echo text with custom color in nusll
<img width="885" alt="截屏2022-04-30 12 10 14" src="https://user-images.githubusercontent.com/42509521/166090060-996776fc-8a49-4342-a93d-6f1c26005ca5.png">

**Comment by kubouch** [comment, marked as answer]
Try the `ansi` command.
